Domains/4-Application/OpenSite/OpenSite.ecschema.xml

```diff
@@ -928,4 +929,13 @@
         </Target>
     </ECRelationshipClass>
 
+    <ECRelationshipClass typeName="AuthoringElementDrivesPlanElement" modifier="None" strength="referencing">
```

Comment:
From BIS naming guidelines: _"The relationship name should indicate the multiplicity"_, see [guidelines](https://www.itwinjs.org/bis/guide/naming-guidelines/rules-and-recommendations/)

This should be _"AuthoringElementDrivesPlanElement**s**"_